### ISSUE TITLE
Change logits to Probs in MetaQA

### ISSUE THREAD

**Issue by HaritzPuerto** [UKP-SQuARE/square-core#450]
https://github.com/UKP-SQuARE/square-core/blob/18f1e8d70127f2f324723d278c7aa487b87b679f/model-inference/model_inference/tasks/inference/metaqa.py#L129

Need to change logits to probs

```
# Run model
logits = self.metaqa_model(input_ids, token_ids, attention_masks, ans_sc).logits
# Get Probabilities
probs = torch.nn.functional.softmax(logits, dim=2) # THIS IS THE NEW LINE!!
# Get predictions
list_preds = self._get_predictions(
    probs.detach().numpy(), request.input_predictions, k=request.top_k
)
```
